pkg/cmd/cli/cli_changefeed_helper.go

```diff
@@ -178,9 +178,9 @@ func sendOwnerAdminChangeQuery(ctx context.Context, etcdClient *etcd.CDCEtcdClie
 	}
 
 	resp, err := httpClient.PostForm(url, map[string][]string{
-		cdc.APIOpVarAdminJob:           {fmt.Sprint(int(job.Type))},
-		cdc.APIOpVarChangefeedID:       {job.CfID},
-		cdc.APIOpForceRemoveChangefeed: {forceRemoveOpt},
+		api.APIOpVarAdminJob:           {fmt.Sprint(int(job.Type))},
```

Comment:
```suggestion
		api.OpVarAdminJob:           {fmt.Sprint(int(job.Type))},
```
May be we can remove the prefix `API` from these exported constant.